src/checkPublication.ts

```diff
@@ -204,16 +204,17 @@ export function checkPublication(
       config.FailureTargetAddresses
     );
 
-  return getRedirect(config.ManifestURL)
-    .then(testRedirect)
-    .then(() => getLogs(logGroupName).then(checkLogs))
-    .then(() =>
-      getS3Objects(
-        config.KindleBucket,
-        `${config.Stage}/${config.Today}/${currentHourString()}`
+  return getLogs(logGroupName)
+      .then(checkLogs)
```

Comment:
We discovered on Saturday night that the `checkLogs` function can reject with an errors object, which is converted to a string in `sendFailureEmail`. While we're here could we either a) make `checkLogs` not reject with an object, or b) make `sendFailureEmail` detect what it has been given to output?

Reply:
Resolved by passing in error message instead of error object. Tested on PROD successfully.